json-endpoints/jetpack/class.jetpack-json-api-themes-install-endpoint.php

```diff
@@ -60,6 +70,14 @@ protected function validate_themes() {
 			}
 
 			if ( wp_endswith( $theme, '-wpcom' ) ) {
+				$skip_download_filter_result = apply_filters( 'jetpack_wpcom_theme_skip_download', false, $theme );
```

Comment:
I think we should move this filter outside of the `-wpcom` suffix check, not because I think we will ever need it that way, but because the other two filters are not inside `-wpcom` checks, which makes this filter inconsistent.